Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -569,11 +575,15 @@ private void ALMRadioButton_Checked_Changed(object sender, RoutedEventArgs e)
                 //Bind again as we changed the AlmConfig object
                 Bind();
 
-                StyleRadioButtons();
-
+                //clear items
+                //DomainComboBox.Items.Clear();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6008688)